### ISSUE TITLE
Single line comments should start with capital letters

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#225]
# Rule request

## Thesis

There are several ways to start a single line comment:

```python
# Example 1

# example 2
```

We should forbid the second way. And enforce single line comments to start with capital letters.

I need to specify several things here:

```python
# This is perfecly
# fine if you ask me
```

Multiline comments may have lines starting with lowercase letters.

```python
# 4 rules:

# @ is allowed:
```

Comments can start with numbers and chars.

```python
print()  # inline comment
```

Inline comments are out of scope of this rule.

## Reasoning

This is done for consistency reasons.

**Comment by son-vyas** [comment]
Is this issue still open? Else I can give it a try

**Comment by sobolevn** [comment]
@son-vyas you are welcome!

**Comment by FloMko** [comment]
@son-vyas  i will make this more faster=)

**Comment by geoc0ld** [comment]
@sobolevn can you help? I have a problem with detect multiple comments.

 I am insert my code into 'comments.py' -> 'WrongCommentVisitor', the 'TokenInfo' provide access only to  comment line and 'exact_type', but i can detect  a line number or file name. Also i can't find the information about previous line in the file. 

I am not sure that 'WrongCommentVisitor' is the best place for my code, but i not found the other way for solution.

**Comment by sobolevn** [comment]
Too complex.
